ceph/ceph_command.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+ceph_command() {
+  operator_pod=$(kubectl get pod -l app=rook-ceph-operator -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+  echo "${*}"
+  kubectl -n rook-ceph exec "$operator_pod" -- /bin/bash -c "${*} --conf=/var/lib/rook/rook-ceph/rook-ceph.config"
```

Comment:
```suggestion
  kubectl -n rook-ceph exec "$operator_pod" -- /bin/bash -c "${*} --conf=/var/lib/rook/rook-ceph/rook-ceph.config"
```
I have a question regarding the config file passed here, this config file change in the case of openshift cluster?

Reply:
The only thing I would expect to change in this path is the namespace. I don't expect any difference for openshift.

Reply:
okay, I'll just do that in follow up pr instead of hard coded to namespace based.

Reply:
The namespace might be different, then `rook-ceph.config` will be `<namespace>.config`.

Reply:
> The namespace might be different, then `rook-ceph.config` will be `<namespace>.config`.

yes, so in follow up PR I'll have read namespace(will have `rook-ceph` default) and will change this accordingly.